openkfw/smartenergy.goecharger
# Rename the integration and added github templates

## Description

Renamed the integration and added github templates.

## Test

- run `start-dev.sh`
- open dashboard in a browser
- everything should look and work as before, no errors and inputs correctly changing the data

- check if I forgot to rename something or renamed something by a mistake
